_articles/evaluation_queues.md

```diff
@@ -2,116 +2,43 @@
 
 title: Evaluation Queues
 layout: article
-excerpt: An queue accepts submission of Synapse entities for evaluation. 
+excerpt: A Synapse Queue contains user submitted models/files to execute for challenges and benchmarking.
 category: reproducible-research
 ---
 
 An Evaluation queue allows for people to submit Synapse `Files`, `Docker` images, etc. for evaluation. They are designed to support open-access data analysis and modeling challenges in Synapse. This framework provides tools for administrators to collect and analyze data models from Synapse users created for a specific goal or purpose.
 
 ## Create an Evaluation Queue
 
-To create a queue, you must first create a Synapse `Project`. To create a Synapse Project, follow the instructions on the [Project and Data Management]({{ site.baseurl}}{% link _articles/getting_started.md %}#making-and-managing-projects-in-synapse) page. An Evaluation queue can take several parameters that you can use to customize your preferences. The minimum requirements to create a queue are:
+To create a queue, you must first create a Synapse `Project`. To create a Synapse Project, follow the instructions on the [Project and Data Management]({{ site.baseurl}}{% link _articles/getting_started.md %}#making-and-managing-projects-in-synapse) page. 
 
-* name – Unique name of the evaluation
-* description – A short description of the evaluation
-* contentSource – Synapse Project associated with the evaluation
-* submissionReceiptMessage – Message to display to users upon submission
-* submissionInstructionsMessage – Message to display to users detailing acceptable formatting for submissions.
+Once you've created your project, navigate to it and add `/challenge` to the url (e.g. www.synapse.org/#!Synapse:syn12345/challenge). Click **Tools** in the right corner and select **Add Evaluation Queue**.
```

Comment:
Could the steps also be 
Select the Challenge tab, click **Challenge Tools*** in the right corner and select **Add Evaluation Queue** ?

Reply:
The challenge tab is hidden unless you are an admin + there is a challenge object associated with it.

Reply:
ah interesting - I guess this was a UI update I missed. 

Ok I agree leaving the wording as **Tools** is appropriate.

What permissions do you need to make a challenge? (e.g. if I'm not admin on a project, enter "/challenge" into the URL, I see **Challenge Tools** with no options.)

Reply:
You need to be an Admin of the project to create a challenge